client/app

```diff
@@ -0,0 +1 @@
+./flutter
```

Comment:
I know we're trying to keep things consistent with `app` and `app-en`, but it feels a little weird to have a symlink here... maybe change `app-en` to `flutter-en`? Or, we (eventually) just move the entire `flutter` dir to `app` (coordinated in advance to minimize the open PRs that will have issues with the files moving)

Reply:
Using the symlink means we can do the right thing and call it app and app_en everywhere (workflows, the pubspec paths, etc.), except purely in the folder.

I've filed https://github.com/WorldHealthOrganization/app/issues/1183 to remove flutter once and for all.